src/input.c

```diff
@@ -1088,13 +1088,12 @@ void process_mouse_over(int x, int y)
 				if (subWindow == NULL)
 					break;
 
-				ebx = ebx & 0xFFFFFF00;
 				edi = cursorId;
 				esi = (int)subWindow;
 				// Not sure what this is for, no windows actually implement a handler
 				// RCT2_CALLFUNC_X(subWindow->event_handlers[WE_UNKNOWN_0E], &eax, &ebx, &ecx, &edx, &esi, &edi, &ebp);
 				cursorId = edi;
-				if ((ebx & 0xFF) != 0)
+				if (0)
```

Comment:
Don't remove this. If we ever work out what WE_UNKNOWN_0E does we might need this. ebx is modified by WE_UNKNOWN_0E.